src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -171,7 +171,11 @@ class BehandlingService(
             vedtakDato = SporbarUtils.now(),
         )
 
-        behandlinghistorikkService.opprettBehandlingshistorikk(behandlingId, BEHANDLING_FERDIGSTILT)
+        behandlinghistorikkService.opprettBehandlingshistorikk(
+            behandlingId = behandlingId,
+            steg = BEHANDLING_FERDIGSTILT,
+            historikkHendelse = null,
```

Comment:
denne kan være implisitt `null` for min del, men også greit å sette den direkte.